components/ai_chat/resources/page/components/input_box/style.module.scss

```diff
@@ -155,6 +155,10 @@
   --leo-icon-color: var(--leo-color-icon-disabled);
 }
 
+.contentAgentButtonEnabled {
+  --leo-icon-color: var(--leo-color-icon-interactive);
```

Comment:
would be nice to follow-up and change these from leo -> ai-chat

Reply:
Leo here refers to the design system whose code name is leo (before AI Chat became Leo). There is an issue to change that to Nala but it's a multi repo operation.